Packs/VirusTotal/IncidentFields/rulename.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
please consult with @ytauman and @Arsenikr if the incident fields should be in common types